Update AA variable docs for python2 functions

jennyhunter-msft opened the pull request:
- Added descriptions of python2 variable functions
- Added sample of how to utilize python2 functions

tysonn commented:
Kay, you probably want to update the ms.date in this article so it is considered fresh with these updates.

singhkay commented:
Updated the date

jennyhunter-msft committed:
Update AA variable docs for python2 functions

- Added descriptions of python2 variable functions
- Added sample of how to utilize python2 functions
